scripts/clean-package.sh

```diff
@@ -14,7 +14,7 @@ cat "$1" | (
 		[ -f "$entry" ] && rm -f $entry
 	done
 )
-cat "$1" | (
+awk '{a[i++]=$0} END {for (j=i-1; j>=0;) print a[j--]}' "$1" | (
```

Comment:
Hm wouldn't a simple `tac` instead of `cat` work here? Or even `sort -r` which should sort length first in reverse, thus listing subdirs / files before dirs.

Reply:
I wasn't sure if `tac` would be available on all systems (e.g. [macOS](https://unix.stackexchange.com/questions/114041/how-can-i-get-the-tac-command-on-os-x)). I'll change it to use `sort -r`, thanks!